.release-notes/v0.0.1.md

```diff
@@ -7,3 +7,7 @@ Release notes for `v0.0.1`.
 - Initial MVP release with support for `Apply`, `Delete`, `Assert` and `Error` operations
 - Supports configuration from both config file and command line flags
 - Full documentation available at https://kyverno.github.io/chainsaw
+
+## 📚 Docs 📚
+
+- Parallel execution of Test
```

Comment:
This should be in `main.md` (not part of v0.0.1)

Reply:
okay let me fix it